Common/ImageSamplers/itkImageRandomSamplerSparseMask.h

```diff
@@ -100,7 +100,7 @@ class ITK_TEMPLATE_EXPORT ImageRandomSamplerSparseMask : public ImageRandomSampl
 private:
   struct UserData
   {
-    ITK_DISALLOW_COPY_AND_MOVE(UserData);
+    //ITK_DISALLOW_COPY_AND_MOVE(UserData);
 
```

Comment:
Thanks @thewtex. I will just remove this line of code afterwards. (No need to keep it commented out.)

Reply:
- FYI, done: pull request #1220 commit ab9878240f2317ac2242cced573f5b4952f26880